docs/layouts/shortcodes/generated/rest_v1_dispatcher.html

```diff
@@ -1368,7 +1368,8 @@
     "job-accumulators" : {
       "type" : "array",
       "items" : {
-        "type" : "any"
+        "type" : "object",
+        "id" : "urn:jsonschema:org:apache:flink:runtime:rest:messages:JobAccumulatorsInfo:JobAccumulator"
```

Comment:
The schema extraction seems to have become smarter. This is a good thing; an object without any properties better resembles an empty response than `any` (which could be _anything_).